src/asmcnc/apps/systemTools_app/screens/screen_factory_settings.py

```diff
@@ -911,7 +913,7 @@ def close_sw(self, dt):
         sys.exit()
 
     def shutdown_console(self, dt):
```

Comment:
Why have this function in the first place? just call console_utils.shutdown() directly